index.html

```diff
@@ -197,35 +199,41 @@ <h2 class="myself">About me</h2>
 
   <section id="contact" class="site-section section-form text-center">
       <div class="contain">
-        <p id="paragraph2">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        <p class="paragraph7">I'm always interested <br> in hearing about new <br>projects, so if you'd <br>like to chat please <br> get in touch.</p>
         <br><br>
        <form id="form" action="https://formspree.io/f/myyvrjyj" method="post" class="myform" >
         <div class="row">
-          <div class="col-sm-6">
-            <input type="text" name="name" class="form-control" placeholder="Full Name" maxlength="30" required>
+          <div class="row1"> 
+            <div class="col-sm-6">
+              <input type="text" name="name" class="form-control3" placeholder="First Name" maxlength="30" required>
+            </div>
+            <div class="col-sm-6f">
+              <input type="text" name="name" class="form-control2" placeholder="Last Name" maxlength="30" required >
+            </div>
           </div>
+         
           <div class="col-sm-6">
             <input type="email" name="email" class="form-control" placeholder="Email Address" required>
           </div>
```

Comment:
- [ ] Same thing with this section. Kindly look into the unwanted space by the right of the section and remove it.

### Your Design
[your-contact-section.webm](https://user-images.githubusercontent.com/26597432/206870322-c991e505-bf34-41d7-ab66-0257b1fa1f0d.webm)

### Figma Design
![figma-contact-section](https://user-images.githubusercontent.com/26597432/206870336-7fd1d6f4-81d2-424a-98a9-1df5bc1c60f7.png)